forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/DeviceBindAuthenticators.kt

```diff
@@ -146,8 +155,9 @@ internal class BiometricOnly(private val biometricInterface: BiometricHandler,
  * Settings for all the biometric authentication and device credential is configured
  */
 internal class BiometricAndDeviceCredential(private val biometricInterface: BiometricHandler,
-                                   private val keyAware: KeyAware,
-                                   private val isApi30OrAbove: Boolean): DeviceAuthenticator {
+                                            private val keyAware: KeyAware,
+                                            private val isApi30OrAbove: Boolean) :
+    DeviceAuthenticator {
```

Comment:
private val isApi30OrAbove: Boolean) : DeviceAuthenticator - Remove Space
 
like this

private val isApi30OrAbove: Boolean):  DeviceAuthenticator - Remove Space